src/initialize.js

```diff
@@ -89,6 +89,7 @@ import {
   APP_READY, APP_INIT_ERROR,
 } from './constants';
 import configureCache from './auth/LocalForageCache';
+import { config } from 'localforage';
```

Comment:
... a little confused how I even managed to get this in here, but removing shortly....